interface/patient_file/summary/rx_left.php

```diff
@@ -18,6 +37,12 @@
 <span><?php xl('Add','e');?></span></a>
 </td>
 </tr>
+<tr>
+<td>
+<a href="<?php echo $GLOBALS['webroot']?>/interface/weno/drug-drug.php"  target='RxRight' class="css_button" onclick="top.restoreSession()">
+<span><?php xl('Drug-Drug', 'e');?></span></a>
+</td>
+</tr>
```

Comment:
wrap this code block with a administration->globals option, such as Drug Interaction Check (drug_interaction_check)